unityhub.py

```diff
@@ -50,9 +50,9 @@ def main():
         replace_string_pref(root, 'kScriptsDefaultApp', b64_editor),
         replace_string_pref(root, 'kScriptEditorArgs', b64_args),
         replace_string_pref(root, 'kScriptEditorArgs/app/bin/code', b64_args),
-        # Generate project files for built-in and registry packages.
-        edit_pref(root, 'unity_project_generation_flag', 'int',
-                  lambda prev: str(int(prev or '0') | 1 << 2 | 1 << 4))
+        # # Generate project files for built-in and registry packages.
```

Comment:
Would rather just remove this entirely ,since we can always get it back from the git history if needed. Kinda nice to hear it's not actually required for it to work anymore.

Reply:
Allright. I patched it out and force pushed a new revision. Thank you for the comment!